src/Symfony/Component/Security/Http/Authenticator/Passport/Badge/UserBadge.php

```diff
@@ -132,4 +128,16 @@ public function isResolved(): bool
     {
         return true;
     }
+
+    private function validateUserIdentifier(string $userIdentifier): void
+    {
+        if ('' === $userIdentifier) {
+            trigger_deprecation('symfony/security-http', '7.2', 'Using an empty string as user identifier is deprecated and will throw an exception in Symfony 8.0.');
```

Comment:
```suggestion
            trigger_deprecation('symfony/security-http', '7.3', 'Using an empty string as user identifier is deprecated and will throw an exception in Symfony 8.0.');
```
?

Reply:
This is moving the logic that was in the constructor, which is why it kept 7.2.